bug: 'PostgreSQLCompiler' has no attribute 'add_operation'

As turned out during my local refactor, the postgres udf tests are not running, there should be a `test_udf.py` [here](https://github.com/ibis-project/ibis/runs/4808027717?check_suite_focus=true#step:13:43). 

There is a valid error, since the `.add_operation()` method has been moved to the `ExpressionTranslator` base class:

```py
        def _translate_udf(t, expr):
            func_obj = sa.func
            if schema is not None:
                func_obj = getattr(func_obj, schema)
            func_obj = getattr(func_obj, name)

            sa_args = [t.translate(arg) for arg in expr.op().args]

            return func_obj(*sa_args)

>       PostgreSQLCompiler.add_operation(udf_node, _translate_udf)
E       AttributeError: type object 'PostgreSQLCompiler' has no attribute 'add_operation'

ibis/backends/postgres/udf.py:124: AttributeError
```

--- a/.github/workflows/ibis-backends.yml
+++ b/.github/workflows/ibis-backends.yml
@@ -194,7 +194,7 @@ jobs:
       - name: run tests
         env:
           PYTEST_BACKENDS: postgres
-        run: ./ci/run_tests.sh -m "not udf"
+        run: ./ci/run_tests.sh
 
       - name: publish test report
         uses: actions/upload-artifact@v2

--- a/ibis/backends/postgres/__init__.py
+++ b/ibis/backends/postgres/__init__.py
@@ -5,8 +5,8 @@ import sqlalchemy
 
 from ibis.backends.base.sql.alchemy import BaseAlchemyBackend
 
-from . import udf
 from .compiler import PostgreSQLCompiler
+from .udf import udf
 
 
 class Backend(BaseAlchemyBackend):

--- a/ibis/backends/postgres/udf.py
+++ b/ibis/backends/postgres/udf.py
@@ -12,7 +12,6 @@ import ibis.udf.validate as v
 from ibis import IbisError
 from ibis.backends.base.sql.alchemy import to_sqla_type
 from ibis.backends.postgres.compiler import (
-    PostgreSQLCompiler,
     PostgreSQLExprTranslator,
     PostgresUDFNode,
 )
@@ -121,7 +120,7 @@ def existing_udf(name, input_types, output_type, schema=None, parameters=None):
 
         return func_obj(*sa_args)
 
-    PostgreSQLCompiler.add_operation(udf_node, _translate_udf)
+    PostgreSQLExprTranslator.add_operation(udf_node, _translate_udf)
 
     def wrapped(*args, **kwargs):
         node = udf_node(*args, **kwargs)

